src/Taxjar/TaxjarApi.cs

```diff
@@ -299,14 +288,8 @@ public virtual CustomerResponseAttributes CreateCustomer(object parameters)
 
         public virtual CustomerResponseAttributes UpdateCustomer(object parameters)
         {
-            var customerIdProp = parameters.GetType().GetProperty("customer_id") ?? parameters.GetType().GetProperty("CustomerId");
-
-            if (customerIdProp == null)
-            {
-                throw new Exception("Missing customer ID for `UpdateCustomer`");
-            }
+            var customerId = GetCustomerIdFromParameters(parameters);
 
```

Comment:
Extra newline here

Reply:
The new line was definitely intentional, are you saying you don't want a new line here?